app/routes.py

```diff
@@ -1,2 +1,131 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
+from app.route_helpers import validate_model
 
+bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet(name=request_body["name"],
+                        description=request_body['description'],
+                        radius=request_body['radius']
+                        )
+    db.session.add(new_planet)
+    db.session.commit()
+
+    return make_response(f"Planet {new_planet.name} successfully created", 201)
```

Comment:
👀 Note that this is still returning a bare HTML result here. At the least, we should consistently return JSON from all our endpoints (use `jsonify`, or pack the message into aq `dict` like we do in the validation helper).

But here, we might also decide to return the created planet record, as this would give the caller access to the `id` that was created, as well as any default values they may not have included in their request.